qemu/tests/cfg/uefi_boot_from_device.cfg

```diff
@@ -58,6 +58,38 @@
                     bootindex_stg = 0
                 - with_specific_device:
                     boot_dev = "UEFI QEMU QEMU HARDDISK"
+        - UEFI_PXEv4:
+            dev_name = "UEFI PXEv4"
```

Comment:
Is the dev name possibly changed?

Reply:
No, I think it's best to keep it unchanged here, so that the output can be clearly seen in the debug log. Thanks.